resources/js/Layouts/Footer.jsx

```diff
@@ -0,0 +1,129 @@
+import ApplicationLogo from "@/Components/ApplicationLogo";
+import { Link } from "@inertiajs/react";
+
+export default function Footer() {
+    return (
+        <footer className="bg-bohim-snow max-w-screen-2xl mx-auto pt-20">
```

Comment:
please reduce footer padding-top to 12 (pt-12)

Reply:
alright sir, i'll do it immediately